fastlane/Fastfile

```diff
@@ -366,8 +366,13 @@ REPOSITORY_NAME="WordPress-Android"
     app = options[:app]
     android_finalize_prechecks(options)
     configure_apply(force: is_ci)
-    
+    check_translation_progress(
+      glotpress_url: 'https://translate.wordpress.org/projects/apps/android/dev/',
```

Comment:
1. I think it would be wise to also check the translations of the Release Notes GP project too, in addition to the one about in-app strings? wdyt?
2. If so, the URL is different between WP and JP for Release Notes GP projects, so we'd want to use the right one depending on the `app:` parameter used. There are already constants in `APP_SPECIFIC_VALUES` for each of those.

Reply:
Sg. I didn't add it because it's something the release manager usually wants to check constantly during the release cycle, but if you think it can be useful, I have no strong opinions against it.

Reply:
Yeah usually I do check those especially the morning before the submission (in preparation for the submission I do the Friday afternoon) so I can ping #i18n if some are missing. But it couldn't hurt to have that additional, automated check in case the RM forgot, better twice than none 🙂